test/y2storage/callbacks/probe_test.rb

```diff
@@ -29,4 +29,83 @@
     include_examples "general #error examples"
     include_examples "default #error true examples"
   end
+
+  describe "#begin" do
+    it "clears again flag" do
+      subject.begin
+      expect(subject.again?).to be false
+    end
+  end
+
+  describe "#again?" do
+    it "is unset after calling begin" do
```

Comment:
I found a bit confusing to use "unset" here as I would expect `nil` there. If I get logic here.  It is `nil` for newly created object, `false` after begin and `true`/`false` after user decision. So I would use explicit `returns false after calling begin`

Reply:
I consider ```again``` to be a flag and there "unset" and "set" is common. But I can change it.